src/ape/api/compiler.py

```diff
@@ -13,6 +13,9 @@ class CompilerAPI:
     def name(self) -> str:
         ...
 
+    """
+    Retrieve avaiailable compiler versions
+    """
     @abstractmethod
     def get_versions(self, all_paths: List[Path]) -> Set[str]:
         ...
```

Comment:
You put the docstrings inside the method like so (we don't need the `...` when we add a docstring)

```suggestion
    @abstractmethod
    def get_versions(self, all_paths: List[Path]) -> Set[str]:
        """
        Retrieve available compiler versions
        """
```